src/main/java/com/conveyal/r5/util/ResultOrError.java

```diff
@@ -0,0 +1,4 @@
+package com.conveyal.r5.util;
+
+public class ResultOrError {
+}
```

Comment:
This is something I keep meaning to do, a parameterized return type that can hold any type T or an exception. It's so much cleaner than throwing exceptions, and yes it would clean up a lot of things happening on backend/workers where we have to repeatedly poll until it's ready. But clearly still not implemented here, so I'll delete it and make a ticket for this instead.